packages/fxa-auth-server/lib/senders/emails/templates/cadReminderFirst/index.stories.ts

```diff
@@ -37,9 +37,9 @@ const commonPropsWithOverrides = (
     },
   });
 
-export const CadReminderMobile = Template.bind({});
-CadReminderMobile.args = commonPropsWithOverrides();
-CadReminderMobile.storyName = 'User is on mobile device';
+export const CadReminderDefault = Template.bind({});
+CadReminderDefault.args = commonPropsWithOverrides();
+CadReminderDefault.storyName = 'default';
```

Comment:
Oh I'm sorry if I wasn't clear - I think we should use "default" if there's no other states, which is the case for both `verificationReminder` templates, but I don't think is the case for others. I think templates with two states, like this one being mobile and non-mobile, should have a description like how it was previously.

Reply:
I think we moved that mobile state from `cadReminder` template to `postVerify`. I had left a [comment](https://github.com/mozilla/fxa/pull/10120#discussion_r689717790) as well in the PR

Reply:
We actually removed the desktop state from `cadReminders` to `postVerify` template. I had also left a [comment](https://github.com/mozilla/fxa/pull/10120#discussion_r689717790) in that PR

Reply:
Gotcha! I didn't realize we weren't sending `onDesktopOrTabletDevice` down on CAD so there's only one state.